src/ralph/lib/mixins/forms.py

```diff
@@ -94,3 +95,24 @@ def clean(self, value):
 
     def to_python(self, value):
         return value
+
+
+class AssetFormMixin(forms.ModelForm):
+    MODEL_TYPE = None
```

Comment:
IMO ``None`` is not valid value - consider some validation

Reply:
What would you consider sufficient validation? Would checking if MODEL_TYPE is not None be enough?